### ISSUE TITLE
No easy way to exit a code block at the end of a file in text editor.

### ISSUE THREAD

**Issue by hwittenborn** [nextcloud/android#9266]
### Steps to reproduce
1. Type the following into an empty text file (Arrow showing where cursor ends up):

```
A `test sentence`
                ^ (directly to right of backtick)
```

2. Place cursor directly to left backtick (now just to the right of the last character in the code block):

```
A `test sentence`
                ^ (very right of generated codeblock)
```

3. With no spaces after the code block, see that there's no way to escape the code block on the right side.

```
A `test sentence`
                ^ (can't get to right of generated codeblock here)
```
---

Sorry if that's not very descriptive, I can try to describe it differently if it doesn't make sense.

**Comment by AndyScherzinger** [comment]
The Android client opens a webview to the server's web UI, so @juliushaertl can you have a look I guess this issue then needs to be transferred to the text repo, what do you think?

**Comment by juliushaertl** [comment]
Yes, seems a valid bug for the text app and also happens on the web, let me move it over
